nbs/16_shortcuts.ipynb

```diff
@@ -258,44 +262,7 @@
    "execution_count": null,
    "id": "b50d27fb-c343-47e2-a008-7ebdd02e1356",
    "metadata": {},
-   "outputs": [
```

Comment:
For future reference, be careful about clearing outputs and then not rerunning cells, because this means it won't appear in the docs any more. Don't worry about it this time -- I'll do it -- this is just an FYI